dll/win32/oleaut32/typelib.c

```diff
@@ -3696,87 +3696,6 @@ static BOOL TLB_GUIDFromString(const char *str, GUID *guid)
   return TRUE;
 }
 
-struct bitstream
-{
-    const BYTE *buffer;
-    DWORD       length;
-    WORD        current;
-};
-
-static const char *lookup_code(const BYTE *table, DWORD table_size, struct bitstream *bits)
```

Comment:
Out of curiosity, this was not REACTOS-specific code that wasn't marked within `__REACTOS__` guards, was it?